internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1868,4 +1868,15 @@ private ConfigKeys() {
    * Config to control the queue capacity for the thread pool executor used for ssl handshake in servers.
    */
   public static final String SERVER_SSL_HANDSHAKE_QUEUE_CAPACITY = "server.ssl.handshake.queue.capacity";
+
+  /**
+   * Number of threads for online Venice producer. This controls how many messages can be processed at a time
```

Comment:
What is the implication of this? Can the config description be a bit more detailed?

Reply:
This sets the number of threads of `AbstractVeniceProducer#producerExecutor`. It controls how many concurrent write operations can be present. Right now, the default value is set to 10, but I was planning on tuning it through some experimentation